composer.json

```diff
@@ -4,11 +4,10 @@
   "description": "Project template for Drutiny development",
   "minimum-stability": "dev",
   "require": {
-    "drutiny/drutiny": "2.2.*@dev",
+    "drutiny/drutiny": "2.3.x-dev",
```

Comment:
This should use `2.3.*@dev` as this allows users to pin to stable releases or run dev (rather than force dev).